src/plugins/memdump/memdump.cpp

```diff
@@ -114,6 +114,40 @@
 
 #define DUMP_NAME_PLACEHOLDER "(not configured)"
 
+static void save_file_metadata(const drakvuf_trap_info_t* info,
+                               const char* file_path,
+                               size_t dump_size,
+                               addr_t dump_address,
+                               const char* method,
+                               const char* dump_reason,
+                               extras_t* extras)
+{
+    char* file = NULL;
+    if ( asprintf(&file, "%s.metadata", file_path) < 0 )
+        return;
+
+    FILE* fp = fopen(file, "w");
+    free(file);
+    if (!fp)
+        return;
+
+    fprintf(fp, "Method: \"%s\"\n", method);
```

Comment:
This is not as easy to automatically parse as JSON output which is originally available when using `-o json` command line switch. At the moment we just use Python 3 and `json.loads()` in order to parse DRAKVUF logs for interesting information, and I'm afraid here it would be required to apply some sophisticated regular expressions or a custom parser.

Reply:
> This is not as easy to automatically parse as JSON output

I use `kv` output format.

> it would be required to apply some sophisticated regular expressions or a custom parser

This is really easy done with Python (and already used in `filedelete` plugin). Because each line have format `key: value`.

Reply:
@icedevml I think I have replied the issue.